core/src/main/java/com/klaytn/caver/tx/type/TxTypeAccountUpdate.java

```diff
@@ -71,4 +76,45 @@ public AccountKey getAccountKey() {
     public Type getType() {
         return Type.ACCOUNT_UPDATE;
     }
+
+    /**
+     * decode transaction hash from sender to reconstruct transaction with fee payer signature.
+     *
+     * @param rawTransaction signed transaction hash from sender
+     * @return TxTypeAccountUpdate decoded transaction
+     */
+    public static TxTypeAccountUpdate decodeFromRawTransaction(byte[] rawTransaction) {
+        //TxHashRLP = type + encode([nonce, gasPrice, gas, from, rlpEncodedKey, txSignatures])
+        try {
+            byte[] rawTransactionExceptType = KlayTransactionUtils.getRawTransactionNoType(rawTransaction);
+            RlpList rlpList = RlpDecoder.decode(rawTransactionExceptType);
+            List<RlpType> values = ((RlpList) rlpList.getValues().get(0)).getValues();
+
+            BigInteger nonce = ((RlpString) values.get(0)).asPositiveBigInteger();
+            BigInteger gasPrice = ((RlpString) values.get(1)).asPositiveBigInteger();
+            BigInteger gasLimit = ((RlpString) values.get(2)).asPositiveBigInteger();
+            String from = ((RlpString) values.get(3)).asString();
+            String rawAccountKey = ((RlpString) values.get(4)).asString();
+
+            TxTypeAccountUpdate tx
+                    = TxTypeAccountUpdate.createTransaction(nonce, gasPrice, gasLimit, from, AccountKeyDecoder.fromRlp(rawAccountKey));
+
+            RlpList vrs = (RlpList) ((RlpList) (values.get(5))).getValues().get(0);
```

Comment:
What if the sender signature has more than one signature?

Reply:
This part is replaced by `protected void addSenderSignatureData (List <RlpType> signatureRlpTypeList)` after #32 merges. Now, it is a temporary implementation to suit the usability of existing caver-java users.

Reply:
So, This PR will be merged after #32 is merged?

Reply:
i merged it. so this method is replaced.